client/components/Events/index.js

```diff
@@ -114,7 +119,11 @@ class EventPage extends Component {
                       </ul>
 
                       <div className="main-cta">
-                        <a href="#" className="btn btn-lg btn-primary cta">
+                        <a
```

Comment:
Since this isn't a hyperlink, can we change it to a `div` or `p` tag or something else.

Reply:
Ok...noted